pkg/accounting/rollup/rollup.go

```diff
@@ -18,19 +20,24 @@ type Rollup interface {
 type rollup struct {
 	logger *zap.Logger
 	ticker *time.Ticker
-	//TODO:
-	//rollupDB
-	//rawDB
+	db     *dbx.DB
 }
 
-func newRollup(logger *zap.Logger, interval time.Duration) *rollup {
+func newRollup(driver, source string, logger *zap.Logger, interval time.Duration) (*rollup, error) {
```

Comment:
Keep logger as the first param.

Reply:
for future reference, WHY?

Reply:
This part is for consistency, in a similar way how ctx is always the first param. If both are present then `ctx`, `logger` (because linter checks wheter ctx is the first).